mozilla-release/mobile/android/app/src/main/res/layout/global_trackers_fragment.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Copyright notice

Reply:
And please add appropriate prefixes like `ghostery_global_trackers_fragment.xml`